src/main/java/org/geysermc/generator/MappingsGenerator.java

```diff
@@ -1055,7 +1081,8 @@ else if (trimmedIdentifier.contains("big_dripleaf")) {
             stateIdentifier = "minecraft:cobblestone_wall";
         }
 
-        if (bedrockIdentifier.startsWith("minecraft:leaves")) {
+        // todo: maybe remove cherry case when bedrock gets it
+        if (bedrockIdentifier.startsWith("minecraft:leaves") && !trimmedIdentifier.contains("cherry")) {
```

Comment:
Can you put the `cherry` contains check in its own method so we can yeet it for 1.20 or whenever?